test/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "env": {
+    "es6": true,
+    "node": true,
```

Comment:
these extra envs aren't required, they still get inherited from the main `.eslintrc`